api/src/main/java/com/kodemon/api/dto/UserDTO.java

```diff
@@ -28,6 +28,8 @@
     private Set<BadgeDTO> badges = new HashSet<>();
     @JsonProperty(required = false, defaultValue = "[]")
     private List<PokemonDTO> pokemons = new ArrayList<>();
+    @JsonProperty(required = false, defaultValue = "[]")
```

Comment:
This is no longer needed, as REST request don't use this DTO anymore (except PUT).